docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md

```diff
@@ -35,7 +35,9 @@ To use ybm CLI, you need to do the following:
 In addition, to use AWS PrivateLink, you need the following:
 
 - An AWS user account with an IAM user policy that grants permissions to create, modify, describe, and delete endpoints.
-- The Amazon resource names (ARN) of security [principals](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html) to which to grant access to the endpoint.
+- The Amazon resource names (ARN) of security [principals](https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#add-remove-permissions) to which to grant access to the endpoint.
```

Comment:
Nit (feel free to ignore): `security principals` is a term used internally, and, AFAIK, exposed to the end-user is some places.
```suggestion
- The Amazon resource names (ARN) of [security principals](https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#add-remove-permissions) to which to grant access to the endpoint.
```